docs/command-line.md

```diff
@@ -418,6 +421,16 @@ processing of output.
 `--use-colour yes` forces coloured output, `--use-colour no` disables coloured
 output. The default behaviour is `--use-colour auto`.
 
+<a id="test-sharding"></a>
+## Test Sharding
+<pre>--shard-count <#number of shards>, --shard-index <#shard index to run></pre>
```

Comment:
Do you mean something like in the benchmark sections: 
```
> [Introduced](https://github.com/catchorg/Catch2/pull/1844) in Catch2 2.11.2.
```
?

Reply:
Yes, but with the right format so that the placeholder is recognized and updated when release is made. See [contributing docs](https://github.com/catchorg/Catch2/blob/devel/docs/contributing.md#technicalities), version tags.